nsot/models/resource.py

```diff
@@ -281,6 +276,21 @@ def _resource_name(self):
     def _purge_attribute_index(self):
         self.attributes.all().delete()
 
+    @classmethod
+    def get_concrete_field_names(self):
```

Comment:
Since you made this a class method, can you please replace `self` with `cls` in keeping w/ naming convention for classmethods?

Reply:
Done.